Source/KSCrash/Recording/Tools/KSObjC.c

```diff
@@ -310,7 +310,8 @@ static int extractTaggedNSString(const void* const object, char* buffer, int buf
     {
         for(int i = 0; i < copyLength; i++)
         {
-            buffer[i] = (char)(value & 0xff);
+            // ASCII case, limit to bottom 7 bits just in case
+            buffer[i] = (char)(value & 0xf7);
```

Comment:
Are you sure about this? I don't think there's anything restricting utf-8 here, aside from the 7 byte limit.